dblock.go

```diff
@@ -97,7 +99,9 @@ func GetDBlock(keymr string) (dblock *DBlock, raw []byte, err error) {
 
 	// TODO: we need a better api call for dblock by keymr so that API will
 	// retrun the same as dblock-byheight
-	return GetDBlockByHeight(db.Header.SequenceNumber)
+	dblockHold, rawHold, errHold := GetDBlockByHeight(db.Header.SequenceNumber)
```

Comment:
The sequence number is just the dblock height. It's an odd term for it, but was probably done to match the json response for the eblock.

Can you push this sequence number adjustment to inside `GetDBlockByHeight`? This way both calls return the same result whether you call by height or by keymr. When you call `GetDBlockByHeight`, you'll have to set the `wrap.DBlock.SequenceNumber` to the parameter `height`.